cli/src/main/java/com/box/l10n/mojito/cli/command/param/Param.java

```diff
@@ -50,7 +50,7 @@
 
     public static final String FILE_TYPE_LONG = "--file-type";
     public static final String FILE_TYPE_SHORT = "-ft";
-    public static final String FILE_TYPE_DESCRIPTION = "File type (if none provided it will scan for default formats: XLIFF, XCODE_XLIFF, MAC_STRING, ANDROID_STRINGS, PROPERTIES, PROPERTIES_NOBASENAME, RESW, RESX)";
+    public static final String FILE_TYPE_DESCRIPTION = "File type (if none provided it will scan for default formats: XLIFF, XCODE_XLIFF, MAC_STRING, ANDROID_STRINGS, PROPERTIES, PROPERTIES_NOBASENAME, RESW, RESX, PO, XTB)";
```

Comment:
yes, wanted to do it but didn't have time yet. Also the import command needs to be documented.